lib/active_campaign/client.rb

```diff
@@ -30,7 +30,7 @@ class Client
     attr_reader :config
 
     def initialize(conf = {})
-      @config = conf
+      @config = Configuration.new.to_h.merge(conf)
```

Comment:
for me, it doesn't merge conf at all and I always end up with default configuration